web/src/frontend/src/nav-soknad/faktum/SporsmalFaktum.tsx

```diff
@@ -47,7 +47,7 @@ class SporsmalFaktum extends React.Component<Props, {}> {
 			setTimeout(() => {
 				if (
 					this.mounted &&
-					!contains(findDOMNode(this), document.activeElement)
+					!is(findDOMNode(this), document.activeElement)
```

Comment:
Jeg er usikker på om dette fungerer. SporsmalFaktum har ikke mulighet for å fokus (har ikke tabIndex), så da vil det aldri kunne bli satt til activeElement => validering vil aldri bli kjørt.